src/icons/bandicoot-bold-button.component.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { useDocumentExecCommand } from '../use-document-exec-command.hook'
+import { useDocumentQueryCommandState } from '../use-document-query-command-state.hook'
+
+export function BoldButton(props) {
```

Comment:
This is different from the API I proposed in https://github.com/CanopyTax/bandicoot/issues/60#issuecomment-532350358. I think I like keeping the IconButton separate from the BoldIcon component, so that IconButton and the logic within it can be highly reused and single sourced.